app/legacy_lib/httparty/logger/commitchange_logger.rb

```diff
@@ -22,8 +22,7 @@
 # LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-
-module HTTParty
+module Httparty
```

Comment:
Zeitwerk didn't like `HTTParty` so I just renamed it. :shrug:

Reply:
https://api.rubyonrails.org/classes/ActiveSupport/Inflector/Inflections.html#method-i-acronym